tests/ctest/test_opmphm_vheap.c

```diff
@@ -0,0 +1,272 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @copyright BSD License (see doc/COPYING or http://www.libelektra.org)
+ */
+
+#include <tests_internal.h>
+
+int maxcomp (void * a, void * b)
```

Comment:
why not static? A very short description would not hurt, even if it is only for unit test.